libs/designer-ui/src/lib/dropdown/index.tsx

```diff
@@ -130,7 +131,7 @@ const getSelectedKey = (options: DropdownItem[], initialValue?: ValueSegment[]):
   if (initialValue?.length === 1 && initialValue[0].type === ValueSegmentType.LITERAL) {
     return (
       options.find((option) => {
-        return option.value === initialValue[0].value;
+        return option.value.toString() === initialValue[0].value.toString();
```

Comment:
case insensitive check?

Reply:
Shouldn't the case be the same if we are comparing the same value that was serialized then deserialized?

Reply:
initialValue[0] should always be a string so we shouldn't need to stringify it; however I think what we can do is instead make option.key a stringified value when initializing the editorViewModel that way we ever only compare a string to a string.